econml/tests/test_dml.py

```diff
@@ -1151,7 +1151,7 @@ def test_groups(self):
             est.fit(y, t, groups=groups)
 
         # test outer grouping
-        est = LinearDML(model_y=LinearRegression(), model_t=LinearRegression(), cv=GroupKFold(2))
+        est = LinearDML(model_y=LinearRegression(), model_t=LinearRegression())
         est.fit(y, t, groups=groups)
 
```

Comment:
```suggestion
assert isinstance(est.splitter, GroupKFold)
```
What about adding something like this. Just to protect against the case where groups isn't actually used under the hood.

Reply:
Though seems like currently we don't save the splitter to our ests as an attribute